tests/should_succeed/sparse_array/proof.json

```diff
@@ -7,15 +7,34 @@
   ],
   "proofs": {
     "M_sparse_array__create": {
-      "vc_create": { "prover": "cvc4@1.8", "time": 0.05 },
-      "vc_from_elem": { "prover": "cvc5@1.0.5", "time": 0.028 },
-      "vc_from_elem'0": { "prover": "cvc5@1.0.5", "time": 0.015 }
+      "vc_create": { "prover": "cvc4@1.8", "time": 0.05 }
     },
     "M_sparse_array__f": {
-      "vc_create": { "prover": "cvc5@1.0.5", "time": 0.033 },
+      "vc_create": { "prover": "cvc5@1.0.5", "time": 0.013 },
+      "vc_create'0": { "prover": "cvc5@1.0.5", "time": 0.025 },
       "vc_f": { "prover": "cvc4@1.8", "time": 0.284 },
       "vc_get": { "prover": "cvc5@1.0.5", "time": 0.018 },
-      "vc_set": { "prover": "cvc5@1.0.5", "time": 0.033 }
+      "vc_get'0": { "prover": "cvc5@1.0.5", "time": 0.021 },
+      "vc_set": { "prover": "cvc5@1.0.5", "time": 0.033 },
+      "vc_set'0": { "prover": "cvc5@1.0.5", "time": 0.025 }
+    },
+    "M_sparse_array__qyi11026919801270832767__bounded_fset_len": {
+      "vc_bounded_fset_len": { "prover": "cvc5@1.0.5", "time": 0.06 }
+    },
+    "M_sparse_array__qyi11026919801270832767__get": {
+      "vc_get": { "prover": "cvc5@1.0.5", "time": 0.051 }
+    },
+    "M_sparse_array__qyi11026919801270832767__lemma_permutation": {
+      "vc_lemma_permutation": { "prover": "cvc5@1.0.5", "time": 0.064 }
+    },
+    "M_sparse_array__qyi11026919801270832767__lemma_permutation_aux": {
```

Comment:
Why is this duplicated in the proof.json file?

Reply:
I removed the proof and re-generated it to only keep fresh goals.